pyvista/core/filters/uniform_grid.py

```diff
@@ -414,3 +414,265 @@ def image_threshold(
+        alg.SetInputDataObject(self)
+        alg.SetCutOff(x_cutoff, y_cutoff, z_cutoff)
+        alg.SetOrder(1)
+        _update_alg(alg, progress_bar, 'Performing Low Pass Filter')
+        output = _get_output(alg)
+        self._change_output_scalars(output, output_scalars_name)
+        return output
+
+    def high_pass(
+        self,
+        x_cutoff,
+        y_cutoff,
+        z_cutoff,
+        order=1,
+        output_scalars_name='ImageScalars',
+        progress_bar=False,
+    ):
+        """Perform a high pass filter in the frequency domain.
+
+        This filter only works on an image after it has been converted to
+        frequency domain by a :func:`UniformGridFilters.fft` filter.
+
+        A :func:`UniformGridFilters.rfft` filter can be
+        used to convert the output back into the spatial
+        domain. This filter attenuates low frequency components.
+        Input and output are in doubles, with two components.
+
+        Parameters
+        ----------
+        x_cutoff : double
+            The cutoff frequency for the x axis.
+
+        y_cutoff : double
+            The cutoff frequency for the y axis.
+
+        z_cutoff : double
+            The cutoff frequency for the z axis.
+
+        order : int, optional
+            The order of the cutoff curve. Given from the equation
+             ``(1 + pow(CutOff/Freq(i, j), 2*Order))``.
+
+        output_scalars_name : str, optional
+            The name of the output scalars. By default this is
+            ``'ImageScalars'``.
+
+        progress_bar : bool, optional
+            Display a progress bar to indicate progress.
+
+        Returns
+        -------
+        pyvista.UniformGrid
+            UniformGrid with the applied high pass filter.
+
+        Examples
+        --------
+        See :ref:`image_fft_perlin_example` for a full example using this filter.
+
+        """
+        self._check_fft_scalars()
+        alg = _vtk.vtkImageButterworthHighPass()
+        alg.SetInputDataObject(self)
+        alg.SetCutOff(x_cutoff, y_cutoff, z_cutoff)
+        alg.SetOrder(order)
+        _update_alg(alg, progress_bar, 'Performing High Pass Filter')
+        output = _get_output(alg)
+        self._change_output_scalars(output, output_scalars_name)
+        return output
+
+    def _change_output_scalars(self, dataset, name):
+        """Modify the name of the output scalars for a FFT filter."""
+        default_name = 'ImageScalars'
+        if name != default_name:
+            pdata = dataset.point_data
+            dataset.point_data[name] = pdata.pop(default_name)
+
+    def _check_fft_scalars(self):
+        """Check for active scalars with two components.
+
+        This is necessary for rfft, low_pass, and high_pass filters.
+
+        """
+        # check for active scalars, otherwise risk of segfault
+        if self.point_data.active_scalars_name is None:
+            raise ValueError('FFT filters require active point scalars')
```

Comment:
I guess we can't use the `set_default_active_scalars` helper because we're looking for 2 components here. Would it make sense to raise this function into `pyvista.utilities.helpers` alongside the scalars/vectors activator utilities?

Reply:
I think this is better left here within this class:
- It relies on class attributes
- It's specific to `UniformGridFilters`